Jenkinsfile

```diff
@@ -332,22 +333,33 @@ pipeline {
               }
             }
           }
-          stage('Release to NuGet') {
-            options { skipDefaultCheckout() }
+          stage('AfterRelease') {
```

Comment:
I renamed the tag as suggested in:
- https://github.com/elastic/apm-agent-java/pull/940#discussion_r354818054

This might be changed in the future once the release is fully automated, but to avoid any misleading let's use another name